modules/util/portal-tools-rest-builder/src/main/resources/com/liferay/portal/tools/rest/builder/dependencies/base_resource_impl.ftl

```diff
@@ -491,6 +490,8 @@ public abstract class Base${schemaName}ResourceImpl
 				return java.math.BigDecimal.ZERO;
 			<#elseif stringUtil.equals(javaMethodSignature.returnType, "java.util.Date")>
 				return new java.util.Date();
+			<#elseif stringUtil.equals(javaMethodSignature.returnType, "jakarta.ws.rs.core.Response")>
```

Comment:
Try to avoid occurrences of jakarta or javax in the templates. We can simplify this code just using the configYAML.getJavaEePackage.